src/app/cli/src/cli_parser.rs

```diff
@@ -324,6 +324,10 @@ pub fn cli() -> Command {
                         Delete a local dataset:
 
                             kamu delete my.dataset
+
+                        Delete a local datasets matching pattern:
```

Comment:
`a` is an article used only with singular countables nouns, so `delete a dataset`, but `delete datasets`